pg_store_plans.c

```diff
@@ -1459,6 +1459,7 @@ pg_store_plans_internal(FunctionCallInfo fcinfo,
 		int			i = 0;
 		int64		queryid      = entry->key.queryid;
 		int64		planid       = entry->key.planid;
+		int64		zero		 = 0L;
```

Comment:
I think it would be better to use UINT64CONST(0).

Reply:
Right. But it is no longer used.